project/ScalaVersion.scala

```diff
@@ -1,10 +1,10 @@
 sealed abstract class MajorScalaVersion
-case object Scala29 extends MajorScalaVersion
 case object Scala210 extends MajorScalaVersion
+case object Scala211 extends MajorScalaVersion
 
 object ScalaVersion {
   def v(implicit sv: String) =
-    if(sv startsWith "2.9.") Scala29
-    else if(sv startsWith "2.10.") Scala210
+    if(sv startsWith "2.10.") Scala210
```

Comment:
Go ahead; it was just a hack due to a change in the way binaries were marked in 2.9 vs 2.10 IIRC.